src/main/java/seedu/duke/command/ViewCommand.java

```diff
@@ -31,15 +31,15 @@ public void execute(String userInput) {
                 System.out.println("Taken modules: ");
                 printModuleList(moduleList.getTakenModuleList());
                 System.out.println("Planned modules: ");
-                printModuleList(moduleList.getToBeTakenModuleList());
+                //printModuleList(moduleList.getToBeTakenModuleList());
```

Comment:
I think you can remove the commented code as it does not value add to the program.